test/unix/main.ml

```diff
@@ -224,33 +204,35 @@ end
 (* Tests of read-only indices *)
 module Readonly = struct
   let readonly () =
-    let Context.{ rw; clone; _ } = Context.empty_index () in
+    let* Context.{ rw; clone; tbl; _ } = Context.with_empty_index () in
     let ro = clone ~readonly:true () in
-    Hashtbl.iter (fun k v -> Index.replace rw k v) main.tbl;
```

Comment:
Unrelated change to get rid of an unnecessary `main` index instance here. Fortunately we're past those days :slightly_smiling_face: